docs/Reference/RPC/commandsRPC.md

```diff
@@ -125,6 +126,73 @@ Common Maintenance Subcommand Options:
 
 <br>
 
+#### addwifisettings
+
+Configure wireless 802.1x settings of an existing, activated AMT device by passing credentials and certificates directly to AMT rather than through RPS/EA/RPC.
```

Comment:
Need to add a note somwhere if the addwifisettings call fails, the certs that were added will be rolled back allowing a retry attempt.